### PR TITLE
edit cmd/cloud-controller-manager/nodeipamcontorller.go  to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/cloud-controller-manager/nodeipamcontorller.go to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/cloud-controller-manager/nodeipamcontroller.go [hunk 1]
```diff
@@ -79,14 +79,14 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 	if len(strings.TrimSpace(nodeIPAMConfig.ServiceCIDR)) != 0 {
 		_, serviceCIDR, err = net.ParseCIDR(nodeIPAMConfig.ServiceCIDR)
 		if err != nil {
-			klog.Warningf("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
+			klog.InfoS("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
```
Comment (cheftako) [inline cmd/cloud-controller-manager/nodeipamcontroller.go:82]: Please take a look at https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/. Switching to structured logs requires adding field names to the list of parameters.

**Comment by yangjunmyfm192085** [comment]
/wg structured-logging
@LinshanYu Could you take a look?

**Comment by pacoxu** [comment]
dup with #100672
/close 
as #100672 was merged.

**Pull request closed without merging**